infrastructure/modules/keyvault/variables.tf

```diff
@@ -0,0 +1,23 @@
+variable "resource_group_name" {
+  description = "Resource group name"
+}
+variable "location" {
+  description = "Resource group location"
+}
+variable "environment" {
+  description = "Environment name to use as a tag"
+}
+variable "name" {
+  description = "Name of the keyvault"
+}
+variable "access_CIDR" {
+  description = "CIDR range to whitelist for access to the keyvault"
+}
+
+variable "authorised_person_ids" {
+  description = "IDs of objects (people etc) to associate access policies for"
```

Comment:
maybe rather than calling people objects, this could say _persons (incl scripted users)_, if my understanding is correct here

Reply:
All people are objects to me!
I updated wording with suggested text.